modules/local/enchantr/report_file_size.nf

```diff
@@ -22,8 +22,12 @@ process REPORT_FILE_SIZE {
     path "file_size_report/tables/log_data.tsv", emit: table
 
     script:
+    def all_logs = logs.join('\n')
+    def logs_file = workDir + '/logs.txt'
+    logs_file = file(logs_file)
+    logs_file.text = all_logs
```

Comment:
was there a problem with the log files? I'm not sure it's the best solution to specify full paths as it can have issues when running in clusters with scratch dirs, etc.